addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/model/OshiSysteminfo.java

```diff
@@ -69,18 +82,15 @@ public OshiSysteminfo() {
         cpu = hal.getProcessor();
         sensors = hal.getSensors();
 
+        // Static objects, should be recreated on each request. In OSHI 4.0.0. it is planed to change this mechanism -
```

Comment:
Great that you left the comment here as well, this is extremely important !

I hope that the OSHI developers will soon solve the problem with this inconsistency.

Please add a TODO as well with information what should be updated in this class, when the OSHI issue is solved.